src/services/kratos/auth-phone-no-password.ts

```diff
@@ -144,7 +145,7 @@ export const AuthWithPhonePasswordlessService = (): IAuthWithPhonePasswordlessSe
     const adminIdentity: AdminUpdateIdentityBody = {
       ...identity,
       credentials: { password: { config: { password } } },
-      state: "active",
+      state: identity.state as IdentityState, // FIXME? type bug from ory library?
```

Comment:
Or if it's not a bug and `undefined` is possible, we can check for it just above this and throw an error if undefined. This may be safer if we can't confirm whether `identity.state` absolutely can't be `undefined` as the type suggests.